db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+# Silly hyrax stuff.... :(
```

Comment:
Hyrax has a bunch of stuff built around admin sets and permission templates. Majority of this stuff assumes there is already an default admin set with a default permission template in your app. Which they do not provide? So I am seeding on here so hyrax actually functions and doesn't just 500.

Will need to run `rake db:setup` or `rake db:seed` to seed your DB with this.